src/common/footer/index.tsx

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import Link from "next/link";
+
+import { LINK_LIST, SNS_LIST } from "./constants";
+import S from "./Footer.module.scss";
+
+const Footer = () => {
+  return (
+    <footer className={S.container}>
+      <div className={S.items}>
+        <span className={S.copyright}>©codeit - 2023</span>
+        <div className={S.links}>
+          {LINK_LIST.map((link) => {
+            return (
+              <Link key={link.id} href={link.link} className="link">
+                {link.name}
+              </Link>
+            );
+          })}
```

Comment:
구조분해할당한다면 더 깔끔할지도!
그리고 유니크한 값이라면 key값에 id 대신 사용해도 됩니다!
```suggestion
          {LINK_LIST.map(({ link, name }) => {
            return (
              <Link key={name} href={link} className="link">
                {name}
              </Link>
            );
          })}
```